### DISCUSSION TITLE
Pastas and non-stationary time series

### DISCUSSION THREAD

**Discussion by IsaDio** [pastas/pastas#322] [Q&A] [answered]
Greetings!

I have been trying to calibrate a PASTAS model to a groundwater head time series controled by climate only.
Historical rainfall (and CDFM) clearly shows that the last 10-15 years have been wetter than average, which translates to an increase in heads over this period. I have been trying to replicate that response in PASTAS without much success.

I know that ARMA models are not suited for non-stationary time series. Am I correct to assume the same for PASTAS or am I missing something?

Thank you for your help. 
Isabelle

**Comment by martinvonk** [comment, marked as answer]
Hi @IsaDio,

Linear trends can be added to a Pastas time series by using `ps.LinearTrend` as a stressmodel. So your time series do not have to be stationary.
